AcceleratorSampleApp/app/src/main/java/com/tokbox/android/accelerator/sample/MainActivity.java

```diff
@@ -248,35 +248,33 @@ public boolean onCreateOptionsMenu(Menu menu) {
     @Override
     public void onRequestPermissionsResult(final int permsRequestCode, final String[] permissions,
                                            int[] grantResults) {
-        switch (permsRequestCode) {
-            case 200:
-                mVideoPermission = grantResults[0] == PackageManager.PERMISSION_GRANTED;
-                mAudioPermission = grantResults[1] == PackageManager.PERMISSION_GRANTED;
-                mReadExternalStoragePermission = grantResults[2] == PackageManager.PERMISSION_GRANTED;
-                mWriteExternalStoragePermission = grantResults[3] == PackageManager.PERMISSION_GRANTED;
-
-                if (!mVideoPermission || !mAudioPermission || !mReadExternalStoragePermission || !mWriteExternalStoragePermission) {
-                    final AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
-                    builder.setTitle(getResources().getString(R.string.permissions_denied_title));
-                    builder.setMessage(getResources().getString(R.string.alert_permissions_denied));
-                    builder.setPositiveButton("I'M SURE", new DialogInterface.OnClickListener() {
-                        @Override
-                        public void onClick(DialogInterface dialog, int which) {
-                            dialog.dismiss();
-                        }
-                    });
-                    builder.setNegativeButton("RE-TRY", new DialogInterface.OnClickListener() {
-                        @Override
-                        public void onClick(DialogInterface dialog, int which) {
-                            dialog.dismiss();
-                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-                                requestPermissions(permissions, permsRequestCode);
-                            }
+        if (permsRequestCode == 200) {
```

Comment:
Can this be a constant.

i mean it is confusing to have a constant with name "permsRequestCode" and validate different variable with same name against constant value. 

Note:  rename "permsRequestCode" to PERMISSION_CODE 

Example:

```
public void onRequestPermissionsResult(final int requestCode, final String[] permissions,
                                           int[] grantResults) {	                                           int[] grantResults) {
        if (requestCode == PERMISSION_CODE ) {
....
```

Reply:
done